### ISSUE TITLE
Translations are not working

### ISSUE THREAD

**Issue by AsavarTzeth** [flathub/com.github.wwmm.pulseeffects#1]
Some apps seem to have issues with translations when they are packaged as a Flatpak.

1. Verified that the required Locale extensions are indeed installed.
2. Verified that distribution packages, of the latest version, does not have this issue.

Apps that experience this issue:
- [Feedreader](https://github.com/flathub/org.gnome.FeedReader)
- [PulseEffects](https://github.com/flathub/com.github.wwmm.pulseeffects)

Things like GtkFileChooseDialog and On/Off buttons work fine. But the application specific strings remain untranslated.

Not sure how to fix this and I cannot find any documentation on it.

**Comment by TingPing** [comment]
If it is happening to multiple applications maybe open an issue against flatpak/flatpak

**Comment by AsavarTzeth** [comment]
Well I did consider that, but I am uncertain if it is something I have done wrong or not. Most applications work as expected. These two were the only ones I have found so far to be affected.

There is one strange difference though. While both of have the same issue, gnome-software does state that FeedReader supports my language, while PulseEffects does not.

But sure, I can open an issue against flatpak/flatpak as well. If nothing else it might get more responses and ideas on where the issue actually is.
